### PR TITLE
Details

### PR DESCRIPTION
**Pull request by srobert99**
#### Description
Created a new fragment, it contains al the data about any freak selected in freaks-screen. The fragment is working properly tablet and phone screens. The fragment is working properly on tablets and phones (landscape/portrait)

### CODE REVIEW

File: app/src/main/kotlin/com/agilefreaks/freaks_catalog/MainActivity.kt [hunk 1]
```diff
@@ -2,15 +2,27 @@ package com.agilefreaks.freaks_catalog
...
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         setupNavigation()
+
+        val navHostFragment = supportFragmentManager
+            .findFragmentById(R.id.nav_host_container) as NavHostFragment
+        // Instantiate the navController using the NavHostFragment
+        navController = navHostFragment.navController
+        // Make sure actions in the ActionBar get propagated to the NavController
+        setupActionBarWithNavController(navController)
```
Comment (cipry13) [inline app/src/main/kotlin/com/agilefreaks/freaks_catalog/MainActivity.kt]: Some lines are duplicated with the setupNavigation() below

File: features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreaksFragment.kt [hunk 2]
```diff
@@ -26,14 +27,14 @@ class FreaksFragment : Fragment() {
 
     private lateinit var viewBinding: FragmentFreaksBinding
 
+    @SuppressLint("InflateParams")
```
Comment (cipry13) [inline features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreaksFragment.kt]: SuppressLint?

File: features/freaks/src/main/res/drawable/profile_pick_background.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    <corners android:radius="10dp" />
+
+
+    <solid android:color="@android:color/white" />
+
+    <padding
+        android:bottom="3dp"
+        android:left="3dp"
+        android:right="3dp"
+        android:top="3dp"/>
```
Comment (cipry13) [inline features/freaks/src/main/res/drawable/profile_pick_background.xml:13]: Harcoded values

File: features/freaks/src/main/res/layout/bottom_sheet_dialog.xml [hunk 4]
```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (cipry13) [inline features/freaks/src/main/res/layout/bottom_sheet_dialog.xml]: Isn't this Andreea's old code?

File: features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreakDetailsFragment.kt [hunk 5]
```diff
@@ -4,21 +4,44 @@ import android.os.Bundle
...
+        viewBinding =
+            DataBindingUtil.inflate(inflater, R.layout.fragment_freak_details, container, false)
+        setViews()
         return viewBinding.root
     }
+
+    private fun setViews() {
+        arguments?.let {
+            val freak = it.getSerializable("freak") as Freak
+            activity
```
Comment (cipry13) [inline features/freaks/src/main/kotlin/com/agilefreaks.freaks_catalog.features.freaks/FreakDetailsFragment.kt:30]: useless line

**Comment by srobert99** [comment]
[Android: Freaks Screen - Details Screen](https://trello.com/c/zdOYA1fX/39-android-freaks-screen-details-screen)

**Pull request closed without merging**